<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="a342637d-0d41-42d4-889d-1d0d1c290769" name="brsatrom@BRSATROM1 2010-07-14 13:55:34" runUser="NORTHAMERICA\brsatrom" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="c614c05c-220d-43eb-8170-0fbece45ac3e">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="brsatrom_BRSATROM1 2010-07-14 13_55_34">
      <DeploymentItem filename="C:\Development\MVC\RazorHelpers\StackOverflow\bin\Debug\StackOverflow.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2010-07-14T13:55:34.3619382-05:00" queuing="2010-07-14T13:55:44.3889408-05:00" start="2010-07-14T13:55:44.6849704-05:00" finish="2010-07-14T13:58:13.4568461-05:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="CanObtainUserIdFromName" storage="c:\development\mvc\razorhelpers\razorhelpertests\bin\debug\razorhelpertests.dll" id="2950b7fc-64ab-b33e-3656-c72745a251bc">
      <Execution id="1a4d3219-ce95-4195-8cd0-089147378ac4" />
      <TestMethod codeBase="C:/Development/MVC/RazorHelpers/RazorHelperTests/bin/Debug/RazorHelperTests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="RazorHelperTests.StackOverFlowTests, RazorHelperTests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="CanObtainUserIdFromName" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="2950b7fc-64ab-b33e-3656-c72745a251bc" executionId="1a4d3219-ce95-4195-8cd0-089147378ac4" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="1a4d3219-ce95-4195-8cd0-089147378ac4" testId="2950b7fc-64ab-b33e-3656-c72745a251bc" testName="CanObtainUserIdFromName" computerName="BRSATROM1" duration="00:02:26.1904119" startTime="2010-07-14T13:55:44.8629882-05:00" endTime="2010-07-14T13:58:12.8597864-05:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="1a4d3219-ce95-4195-8cd0-089147378ac4">
      <Output>
        <ErrorInfo>
          <Message>Test method RazorHelperTests.StackOverFlowTests.CanObtainUserIdFromName threw exception: 
Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: ‹. Line 1, position 1.</Message>
          <StackTrace>    at Newtonsoft.Json.JsonTextReader.ParseValue(Char currentChar) in d:\Development\Releases\Json\Working\Src\Newtonsoft.Json\JsonTextReader.cs:line 620
   at Newtonsoft.Json.JsonTextReader.Read() in d:\Development\Releases\Json\Working\Src\Newtonsoft.Json\JsonTextReader.cs:line 277
   at Newtonsoft.Json.Linq.JObject.Load(JsonReader reader) in d:\Development\Releases\Json\Working\Src\Newtonsoft.Json\Linq\JObject.cs:line 252
   at Newtonsoft.Json.Linq.JObject.Parse(String json) in d:\Development\Releases\Json\Working\Src\Newtonsoft.Json\Linq\JObject.cs:line 280
   at RazorHelpers.StackOverflow.Flair.GetId(String username)
   at RazorHelperTests.StackOverFlowTests.CanObtainUserIdFromName() in C:\Development\MVC\RazorHelpers\RazorHelperTests\StackOverFlowTests.cs:line 15
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>